src/patternfly/components/Wizard/wizard.scss

```diff
@@ -478,6 +509,26 @@
   flex-grow: 1;
 }
 
+.#{$wizard}__nav-link-main {
+  display: flex;
+  flex-grow: 1;
+  justify-content: space-between;
+  padding-block-start: var(--#{$wizard}__nav-link--PaddingBlockStart);
+  padding-block-end: var(--#{$wizard}__nav-link--PaddingBlockEnd);
+  padding-inline-start: var(--#{$wizard}__nav-link--PaddingInlineStart);
+  padding-inline-end: var(--#{$wizard}__nav-link--PaddingInlineEnd);
+  background-color: var(--#{$wizard}__nav-link--BackgroundColor);
+  border: none;
+  border-radius: var(--#{$wizard}__nav-link--BorderRadius);
+}
+
+.#{$wizard}__nav-link-status-icon {
```

Comment:
One thing to note about the font awesome status icons is that the warning-triangle icon is wider than the icons we use for success/danger, so if we put a warning icon by a step name, that step text will be pushed over farther than the step text above/below it. It's not a huge issue since the icons in react are square, and most users will use those, but if a user supplied their own icon that wasn't a square, that may cause an issue.

I wonder if this element should share the shape size we define for the count circles, or we should reconsider using absolute positioning/translate here like it was before. Or possibly use subgrid for the number/status icon, though worth noting the column width would shift for all steps if a wider status icon was added to a step.

<img width="349" alt="Screenshot 2024-04-18 at 5 07 06 PM" src="https://github.com/patternfly/patternfly/assets/35148959/7a3f2844-1e10-46e6-8aa8-06ed44e13021">
<img width="353" alt="Screenshot 2024-04-18 at 5 12 14 PM" src="https://github.com/patternfly/patternfly/assets/35148959/77ba3209-0914-4ea4-9012-7c335717f656">
<img width="342" alt="Screenshot 2024-04-18 at 5 12 22 PM" src="https://github.com/patternfly/patternfly/assets/35148959/078299da-174b-4d50-9610-9b540bba8c15">

Reply:
@mcoker Can we defer this pending a decision about the new icon set?

Reply:
Yep, good plan 👍